scenes/enumerated/problem_11104/problem_11104.py

```diff
@@ -0,0 +1,310 @@
+from manim import LEFT, UP, DOWN, RIGHT
+from manim import FadeOut, Write
+from manim import AnimationGroup, Group, VGroup, ReplacementTransform, Wiggle, Transform, Animation
+from manim import Line, Rectangle, Arrow, MathTex
+from manim import UL, DR, UR, DL, PI
+from manim import RED, GREEN, ORANGE, BLUE_C, YELLOW, PURE_BLUE, WHITE
+
+from movement_problems import CustomMovementScene
+from hanrahashiv import ModifyFormula, FormulaModificationsScene
+from . import text
+
+
+class Problem11104(CustomMovementScene):
+    """Վեց տարբեր քարտերի վրա գրված են 61, 24, 7, 599, թվերը: Այդ քարտերը դասավորելով իրար կողքի
+     կստացվի 8-անիշ թիվ: Այդ եղանակով ստացեք՝
+    ա) հնարավոր ամենամեծ 8-անիշ թիվը,
+    բ) հնարավոր ամենափոքր 8-անիշ թիվը:"""
+    def construct(self):
+        screen_center = [0, 0, 0]
+        condition_point = [0, 3.40, 0]
+        self.add_task_number(text=text.TASK_NUMBER_STR)
+        self.add_plane()
+
+        # def smart_rectangles(qty, position="horizontal")
+        #     Rectangle(height=1.5, width=2)
+
+        # -------------------------- Point 1 ------------------------------- #
+        condition_1 = MathTex(text.CONDITION_1).move_to(condition_point).scale(1.75).shift(RIGHT)
+        self.play(Write(condition_1))
+        self.wait()
+
+        # -------------------------- Point 2 ------------------------------- #
+        rectangle_1 = Rectangle(height=1.5, width=2)
+        rectangle_2 = rectangle_1.copy().next_to(rectangle_1)
+        rectangle_3 = Rectangle(height=1.5, width=1).next_to(rectangle_2)
+        rectangle_4 = Rectangle(height=1.5, width=3).next_to(rectangle_3)
+        rectangles = VGroup(rectangle_1, rectangle_2, rectangle_3, rectangle_4
+                           ).move_to(screen_center).scale(0.6)
```

Comment:
rectangles = VGroup(*[Rectangle(height=1.5 * 0.6, width=2 * 0.6) for _ in range 4]).arrange(RIGHT, buff=0.6).move_to(screen_center)

պետք ա որ սենց ճիշտ աշխատի, ուղղանկյան չափերը հենց սկզբից փոքր ես վերցնում ու next_to-ն էլ 0.6 ա սարքում միանգամից հեռավորությունը

Reply:
մերսի